src/theme.tsx

```diff
@@ -228,7 +229,7 @@ export type ThemeConfig = {
         logEvent: (trackingEvent: TrackingEvent) => Promise<void>;
         eventFormat?: EventFormat;
     };
-    dimensions?: {headerMobileHeight: number};
+    dimensions?: {headerMobileHeight: number; headerDesktopHeight: number};
```

Comment:
as commented in Teams, I wouldn't allow to configure the `headerDesktopHeight`. Use the internally defined value and expose it via context, so you can use it if needed